galsim/detectors.py

```diff
@@ -297,7 +350,9 @@ def quantize(self):
     """
     self.applyNonlinearity(numpy.round)
 
+
 galsim.Image.applyNonlinearity = applyNonlinearity
 galsim.Image.addReciprocityFailure = addReciprocityFailure
```

Comment:
Given that `addReciprocityFailure` has already made into v1.3, shouldn't we
continue to support this routine, at least for a while? We can have a
warning that says that this routine is depreciated and calling
`addReciprocityFailure` could in turn be made call the new routine
`applyReciprocityFailure`.

On Wed, Feb 10, 2016 at 2:04 PM, Mike Jarvis <notifications@github.com>
wrote:

> In galsim/detectors.py
> <https://github.com/GalSim-developers/GalSim/pull/710#discussion_r52506965>
> :
>
> > @@ -297,7 +350,9 @@ def quantize(self):
> >      """
> >      self.applyNonlinearity(numpy.round)
> >
> > +
> >  galsim.Image.applyNonlinearity = applyNonlinearity
> >  galsim.Image.addReciprocityFailure = addReciprocityFailure
>
> Not this code, but "add" also seems wrong to me for reciprocity failure.
> Apply seems more apt to me here as well. It doesn't add some kind of
> "reciprocity failure" feature to the image, it applies the effect of
> reciprocity failure to the ADU values in the image.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GalSim-developers/GalSim/pull/710/files#r52506965>.
>



-- 
Regards

Arun Kannawadi Jayaraman
PhD candidate, Dept. of Physics,
Carnegie Mellon University,
Pittsburgh, PA 15213